src/applications/simple-forms/40-0247/tests/e2e/0247-pmc.cypress.spec.js

```diff
@@ -24,13 +24,13 @@ const awaitFocusSelectorThenTest = () => {
   return ({ afterHook }) => {
     cy.injectAxeThenAxeCheck();
     afterHook(() => {
-      cy.get(v3StepHeaderSelector)
+      cy.get(v3StepHeaderSelector, { timeout: 10000 })
         .should('be.visible')
         .then(() => {
-          // callback to prevent scroll/focus interferences, but
-          // even now field-disabled errors still occur, so must wait a bit.
+          // callback to avoid field-disabled errors, but
+          // even now we must wait a bit!
           // eslint-disable-next-line cypress/no-unnecessary-waiting
-          cy.wait(500);
+          cy.wait(1000);
```

Comment:
can you try keeping this a low number? the main issue was that height/width was 0 which mean it wasn't visible. so line 27/28 was the failure line I believe.